dhis-2/dhis-api/src/main/java/org/hisp/dhis/cache/CacheProvider.java

```diff
@@ -139,4 +139,6 @@
     <V> Cache<V> createDataIntegrityDetailsCache();
 
     <V> Cache<V> createSubExpressionCache();
+
+    <V> Cache<V> createExpressionDimensionItemAnalyticsResponseCache();
```

Comment:
We could make it a bit shorter... like:
`createExpressionDimensionItemAnalyticsCache`